about.html

```diff
@@ -0,0 +1,115 @@
+                 to meet their favourite writers. The 2021 edition will take place in Athens.</p>
+            <p class="about-contact">Please contact us per Email for any further question about IWC 2021</p>
+            <p class="about-mail"><u>IWC2021@athens.org</u></p>
+             
+        </section>                  
+        
+        <section class="venue-decision">
+            <h3 class="decision">The 2021 International Writers' Conference</h3>
+            <hr>
+            <p class="venue">The venue for 2021 International Writers' Conference was decided by a panel of 47 members selected across the world.</p>
+            <ul class="conference-ul">
+                <li class="conference-title">International Writers' Conference 2021</li>
+                <li class="date-and-place">9th - 11th JULY 2021 | Athens, Greece</li>
+            </ul>
+        </section>
+        
+        <section class="past-conference">
+            <h3 class="past-conference-heading">See the past International Writers' Conference</h3>
+            <hr>
+            <p class="past-conference-text">Take a look at the past two International Writers' Conference that took place in Saigon and Sydney</p>
+            <div class="past-conference-image">
+                <div class="saigon-2018">
+                    <div class="conference-background-color">
+                    <p class="past-conference-image-date">2018</p>
+                    <p class="past-conference-image-text">IWC 2018 in Saigon</p>
+                </div>
+               </div>
+                <div class="sydney-2019">
+                    <div class="conference-background-color">
+                       <p class="past-conference-image-date">2019</p>
+                       <p class="past-conference-image-text">IWC 2019 in Sydney</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="partners">
+            <h2 class="partners-title">Sponsors</h2>
+            <ul class="partners-ul">
+                <li class="partners-list"><img src="Assets/mozilla.svg" alt="Mozilla Img"></li>
+                <li class="partners-list"><img src="Assets/google.svg" alt="Google Img"></li>
+                <li class="partners-list"><img src="Assets/airbnb.svg" alt="Naver Img"></li>
+                <li class="partners-list"><img src="Assets/naver.svg" alt="Airbnb Assets"></li>
+            </ul>
+        </section>
+        <footer class="copyrights">
+            <div class="footer-logo"><img class="footer-image" alt="book-icon" src="Assets/book-icon.svg"></div>
+            <div class="credits-container">
+                <h3 class="copyrights-title">International Writers' Conference</h3>
+                <p class="rights-reserved">The International Writers' Conference, 2021. Some rights reserved</p>
+            </div>
+    </footer>
+
+    </main>

```

Comment:
Oops., Let's follow the [Semantic structure](https://www.freecodecamp.org/news/semantic-html5-elements/) `footer` tag should not be inside of a `main` tag.